manim/mobject/geometry/tips.py

```diff
@@ -76,22 +76,41 @@ class ArrowTip(VMobject, metaclass=ConvertToOpenGL):
     .. manim:: ArrowTipsShowcase
         :save_last_frame:
 
-        from manim.mobject.geometry.tips import ArrowTriangleTip,\
-                                                ArrowSquareTip, ArrowSquareFilledTip,\
-                                                ArrowCircleTip, ArrowCircleFilledTip
+        from manim.mobject.geometry.tips import (
+            ArrowTriangleTip,
+            ArrowSquareTip,
+            ArrowSquareFilledTip,
+            ArrowCircleTip,
+            ArrowCircleFilledTip,
+            StealthTip
+        )
```

Comment:
Instead of explicitly importing every arrow tip, we could do something like
```py
        import manim.mobject.geometry.tips as arrow_tips
```
which would then allow us to access all of the tips in the file via the module, e.g., `arrow_tips.ArrowTriangleTip` and so on. Do you think this would improve the example?

Reply:
It seems they don't need to be imported explicitly, they're available from importing * from manim.   However, I have improved the example a little more and squashed the commits.